src/isIsogram.test.js

```diff
@@ -6,4 +6,18 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('should return true if comes an empty string', () => {
+    expect(isIsogram('')).toBeTruthy();
+  });
+
+  it('should return correct value if a non-empty string is received', () => {
+    expect(isIsogram('playgrounds')).toBeTruthy();
+    expect(isIsogram('look')).toBeFalsy();
+  });
+  
+  it('should return a valid string regardless of case', () => {
+    expect(isIsogram('LooK')).toBeFalsy();
```

Comment:
'LooK' doesn't check the case-insensitivity: you have different letters in different cases. You should check the same letters in different cases, e.g. 'LOok'